src/csvcubed/utils/sparql_handler/csvw_inspector.py

```diff
@@ -59,3 +85,17 @@ def get_primary_catalog_metadata(self) -> CatalogMetadataResult:
         raise KeyError(
             f"Could not find catalog metadata in primary graph '{self.primary_graph_uri}'."
         )
+
+    def get_table_schema_properties(self) -> TableSchemaPropertiesResult:
```

Comment:
I think this should be `get_table_info_for_csv_url(self, csv_url: str)....`

Reply:
Done